.github/workflows/CI.yml

```diff
@@ -24,6 +24,12 @@ jobs:
       # Clone the repo and checkout the commit for which the workflow was triggered
       - uses: actions/checkout@v2
 
+      - name: Test integrity of duplicate values
+        shell: bash
+        run: |
+          chmod +x ./scripts/test_duplicate_values.sh
+          ./scripts/test_duplicate_values.sh
+
```

Comment:
#774 checks that the list of files in the Service Worker is consistent with the files in the filesystem. This test is different: it checks that certain duplicated values between `app.js` and `service-worker.js` are consistent with each other. It's a way to ensure a dev doesn't change a value in one place without changing a required corresponding value. It could be a separate PR, but conceptually, for me, it belongs with the changes being made here.

Reply:
OK, sorry for the confusion.
You're right : it should belong to this PR